lib/rules/sotd/stability.js

```diff
@@ -14,8 +14,8 @@ function findSW(candidates, sr) {
         sr.config.longStatus === 'Group Note' ||
         sr.config.longStatus === 'Group Draft Note'
     ) {
-        // Find the sentence of 'Group Notes are not endorsed by W3C nor its Members.'
-        wanted = `${sr.config.longStatus}s are not endorsed by W3C nor its Members.`;
+        // Find the sentence of 'Group Notes are not endorsed by W3C nor its Members.' or 'This Group Note is endorsed by the @@ Group, but is not endorsed by W3C itself nor its Members.'
```

Comment:
you also need to take into account documents that are published by 2 groups

Reply:
A: This Group (Draft )?Note is endorsed by the XXX Working/Interest Group (and XXX Working/Interest Group)?, but is not endorsed by W3C itself nor its Members.

OK